locales/fr-FR/App.json

```diff
@@ -213,6 +213,7 @@
   "(current)": "(actif)",
   "Default address for sending transactions.": "Adresse par défaut pour effectuer des transactions.",
   "Error while fetching data for address {{ hash }}": "Erreur lors de la récupération des données de l'adresse {{ hash }}",
+  "Error while fetching pending transactions for address {{ hash }}": "Error while fetching pending transactions for address {{ hash }}",
```

Comment:
@mvaivre I could use your help translating this :)

Reply:
"Erreur durant la récupération de transactions en attente pour l'adresse {{ hash }}"